pyproject.toml

```diff
@@ -2,6 +2,9 @@
 requires = ["setuptools==75.4.0", "setuptools-scm==8.1.0", "wheel==0.45.0", "auditwheel==6.1.0"]
 build-backend = "setuptools.build_meta"
 
+[project]
+requires-python = ">=3.9"

```

Comment:
`black` [starts reading `[project.requires-python]` ](https://black.readthedocs.io/en/stable/usage_and_configuration/the_basics.html#:~:text=By%20default%2C%20Black%20will%20infer%20target%20versions%20from%20the%20project%20metadata%20in%20pyproject.toml%2C%20specifically%20the%20%5Bproject.requires%2Dpython%5D%20field.%20If%20this%20does%20not%20yield%20conclusive%20results%2C%20Black%20will%20use%20per%2Dfile%20auto%2Ddetection.) from the `pyproject.toml` - hence adding it here.